docs/user/openstack/README.md

```diff
@@ -242,43 +242,50 @@ All the OpenShift nodes get created in an OpenStack tenant network and as such,
 
 This method allows you to attach two floating IP (FIP) addresses to endpoints in OpenShift.
 
-You will need to create two floating IP addresses, one to attach to the API load balancer (lb FIP), and one for the OpenShift applications (apps FIP). Note that the LB FIP is the same floating IP as the one you added to your `install-config.yaml`. The following command is an example of how to create floating IPs:
+A standard deployment uses three floating IP addresses in total:
 
-```sh
-openstack floating ip create <external network>
-```
+1. External access to the OpenShift API
+2. External access to the workloads (apps) running on the OpenShift cluster
+3. Temporary IP address for bootstrap log collection
 
-You will also need to add the following records to your DNS:
+The first two addresses (API and Ingress) need to be created up-front and have the corresponding DNS records resolve to them.
 
-```dns
-api.<cluster name>.<base domain>  IN  A  <lb FIP>
-```
+The third floating IP is created automatically by the installer and will be destroyed along with all the other bootstrap resources. If the bootstrapping process fails, the installer will try to SSH into the bootstrap node and collect the logs.
 
-If you don't have a DNS server under your control, you should add the records to your `/etc/hosts` file.
+##### Create API and Ingress Floating IP Addresses
 
-**NOTE:** *this will make the API accessible only to you. This is fine for your own testing (and it is enough for the installation to succeed), but it is not enough for a production deployment.*
+You will need to create two floating IP addresses, one to attach to the API load balancer (lb FIP), and one for the OpenShift applications (apps FIP). Note that the LB FIP is the IP address you will add to your `install-config.yaml` or select in the interactive installer prompt.
 
-In order to reach the applications running on your worker nodes, you should attach a floating IP to the `ingress-port` at the end of your install. That can be done in the following steps:
+The following command is an example of how to create the floating IPs:
 
 ```sh
-openstack port show <cluster name>-<clusterID>-ingress-port
+openstack floating ip create <external network>
+# => <lb FIP>
+openstack floating ip create <external network>
+# => <apps FIP>
 ```
 
-Then attach the FIP to it:
+**NOTE:** These IP addresses will **not** show up attached to any particular server (e.g. when running `openstack server list`). Similarly, the API and Ingress ports will always be in the `DOWN` state.
 
-```sh
-openstack floating ip set --port <ingress port id> <apps FIP>
-```
+This is because the ports are not attached to the servers directly. Instead, their fixed IP addresses are managed by keepalived. This has no record in Neutron's database and as such, is not visible to OpenStack.
+
+*The network traffic should flow through even though the IPs and ports do not show up in the servers*.
```

Comment:
Let's be bold! "s/should/will"? :grin:

Reply:
I had it there and chickened out :-). Going to put it back.